### ISSUE TITLE
Weights are in CPU format

### ISSUE THREAD

**Issue by akshitamittel** [junyanz/pytorch-CycleGAN-and-pix2pix#254]
After your latest commit, I am getting the following error during train, I am trying to add .cuda() to the models however, is there an alternate fix? Thank you.

```
Traceback (most recent call last):

  File "train.py", line 31, in <module>

    model.optimize_parameters()

  File "/home/amittel/Desktop/pytorch-CycleGAN-and-pix2pix/models/cycle_gan_model.py", line 157, in optimize_parameters

    self.backward_G()

  File "/home/amittel/Desktop/pytorch-CycleGAN-and-pix2pix/models/cycle_gan_model.py", line 124, in backward_G

    self.idt_A = self.netG_A(self.real_B)

  File "/home/amittel/anaconda2/envs/pytorch_sem/lib/python3.5/site-packages/torch/nn/modules/module.py", line 325, in __call__

    result = self.forward(*input, **kwargs)

  File "/home/amittel/anaconda2/envs/pytorch_sem/lib/python3.5/site-packages/torch/nn/parallel/data_parallel.py", line 66, in forward

    return self.module(*inputs[0], **kwargs[0])

  File "/home/amittel/anaconda2/envs/pytorch_sem/lib/python3.5/site-packages/torch/nn/modules/module.py", line 325, in __call__

    result = self.forward(*input, **kwargs)

  File "/home/amittel/Desktop/pytorch-CycleGAN-and-pix2pix/models/networks.py", line 199, in forward

    return self.model(input)

  File "/home/amittel/anaconda2/envs/pytorch_sem/lib/python3.5/site-packages/torch/nn/modules/module.py", line 325, in __call__

    result = self.forward(*input, **kwargs)

  File "/home/amittel/anaconda2/envs/pytorch_sem/lib/python3.5/site-packages/torch/nn/modules/container.py", line 67, in forward

    input = module(input)

  File "/home/amittel/anaconda2/envs/pytorch_sem/lib/python3.5/site-packages/torch/nn/modules/module.py", line 325, in __call__

    result = self.forward(*input, **kwargs)

  File "/home/amittel/anaconda2/envs/pytorch_sem/lib/python3.5/site-packages/torch/nn/modules/conv.py", line 277, in forward

    self.padding, self.dilation, self.groups)

  File "/home/amittel/anaconda2/envs/pytorch_sem/lib/python3.5/site-packages/torch/nn/functional.py", line 90, in conv2d

    return f(input, weight, bias)

RuntimeError: Input type (CUDAFloatTensor) and weight type (CPUFloatTensor) should be the same
```

**Comment by akshitamittel** [comment]
I wasn't getting this error in the commit before.

**Comment by Naruto-Sasuke** [comment]
A quick workaround here https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/issues/253 if you only has one GPU.

**Comment by taesung89** [comment]
We fixed the issue(ea3b646) following @Naruto-Sasuke 's suggestion.
